frontend/app/controllers/Outages.scala

```diff
@@ -3,7 +3,7 @@ package controllers
 import play.api.mvc.Controller
 
 object Outages extends Controller {
-  def maintenanceMessage = CachedAction {
+  def maintenanceMessage = NoCacheAction {
```

Comment:
Ah good point, this page contains static content and is on a static public URL. The dynamic-ness comes from the redirect. I'll remove that change.